packages/dashboard/src/views/Holdings.js

```diff
@@ -101,16 +101,34 @@ const Holdings = ({
       const currAllData = historicQuotes['all']
       const currAllResults = currAllData.results[currAllData.results.length - 1]
 
-      // override current balance
+      // Override cash value (buying power)
       smallStats[0] = {
         ...smallStats[0],
+        value: Math.round(accounts["non-registered-ijzmubrj"].buying_power.amount * 100) / 100
+      }
+
+      // Override securities value
+
+      const positions = accounts["non-registered-ijzmubrj"].positions
+      let securitesValue = 0
+      for (let i=0; i < positions.length; i++) {
```

Comment:
This can be reformatted into a `reduce` check out the looping array functions that are super useful. `map`, `filter` and `reduce`. Generally there is no need for a for loop in modern JS. Also install the prettier vscode extension. I have a formatter file in the repo so when you save it will automagically format everything so we don't clutter PRs with code styling stuff.